### ISSUE TITLE
About the COCO dataset

### ISSUE THREAD

**Issue by DesertsP** [ZHANGDONG-NJUST/CONTA#17]
How to split the training set, validation set and test set? How to get the mask of semantic segmentation? Thanks.

**Comment by ZHANGDONG-NJUST** [comment]
1. COCO used in our work refers to COCO-2014, you can find the corresponding data set on the website.
2. segmentation mask can be obtain by the trained segmentation model in Step-3.
